aisdc/attacks/worst_case_attack.py

```diff
@@ -492,7 +498,7 @@ def _get_attack_metrics_instances(self) -> dict:
         attack_metrics_experiment = {}
         attack_metrics_instances = {}
 
-        for rep in range(len(self.attack_metrics)):
+        for rep, _ in enumerate(self.attack_metrics):
             attack_metrics_instances["instance_" + str(rep + 1)] = self.attack_metrics[
```

Comment:
why the +1 here? The json is entirely for machine consumption, and this will mean all developers will have to +- 1?